modules/artifact/src/test/java/org/mule/runtime/module/artifact/classloader/FineGrainedControlClassLoaderTestCase.java

```diff
@@ -52,6 +61,9 @@ public void usesParentOnlyLookupAndFails() throws Exception
         final ClassLoaderLookupPolicy lookupPolicy = mock(ClassLoaderLookupPolicy.class);
         when(lookupPolicy.getLookupStrategy(TEST_CLASS_NAME)).thenReturn(PARENT_ONLY);
 
+        expected.expect(CompositeClassNotFoundException.class);
+        expected.expectMessage(startsWith("Cannot load class '" + TEST_CLASS_NAME + "': [ERROR" + lineSeparator() + "]"));
```

Comment:
No test is checking that the composite exception contains the different exceptions (from the parent and the child if corresponds in each case)